### DISCUSSION TITLE
Combine Text Detection with Classification from MMClassification

### DISCUSSION THREAD

**Discussion by yCobanoglu** [open-mmlab/mmocr#1810] [Q&A]
Is there a straight forward way to combine these two ? Similr to the decoder, encoder architecture maybe but i would have to be able to use MMClassification Models in my configuration file. I have looked into the docs and code but i am unsure if should try to make it work within MMOCRs framework (i need it for evaluation the models are trained seperately) or if i should code my own evaluation.

**Comment by gaotongxiao** [comment]
Sure - it's straightforward. While we haven't added such a section to the docs, you can refer to [MMDet's docs](https://mmdetection.readthedocs.io/en/latest/tutorials/how_to.html#use-backbone-network-through-mmclassification) for now.

Another example you may be interested in is [Mask R-CNN](https://github.com/open-mmlab/mmocr/blob/9caacc76ee1e3f22b33da2d305128150b5098ab5/configs/textdet/maskrcnn/_base_mask-rcnn_resnet50_fpn.py), where we use [MMDetWrapper](https://github.com/open-mmlab/mmocr/blob/e067ddea2395f4d6dfe7e10922f06cd096bcedb4/mmocr/models/textdet/detectors/mmdet_wrapper.py) that wraps around MMDet's detection model and fits it to text detection context.
